app/component/embeddedSearchGenerator.scss

```diff
@@ -82,6 +82,16 @@
                     z-index: 1;
                 }
             }
+
+            .datetimepicker-container {
+                margin-top: 1rem;
+
+                & fieldset {
+                    margin: 0 !important;
+                    padding: 0 !important;
+                    max-width: 399px !important;
```

Comment:
We should try to avoid using these !important style definitions. I think the issue here is probably that the styles from here leak to the timepicker. We should either make those generic style definitions here that affect all html elements be more specific and only affect certain classes or make some wrapper divs for the other elements in the generator and make the generic styles under those divs so they don't affect the preview.